packages/client/src/components/SortBy/SortBy.styles.css

```diff
@@ -47,16 +47,15 @@
 }
 
 .select-sort-by {
-  appearance: none;
   background: linear-gradient(180deg, #231c16 0%, rgb(35 28 22 / 75%) 100%);
+  background-image: url('../../../public/assets/vectors/vector_arrow_down.svg');
```

Comment:
-webkit-appearance: none;

I suppose with this line we may get rid of default drop down arrow as it is partially visible under used vector

Reply:
and that would be better to remove ":" before <<Sort by >> phrase in SortBy.component

Reply:
probably it will need padding-right: 30px; in .select-sort-by 

try the longest string option

Reply:
Thank you so much @negaren . I literaly change the `.svg` size instead of trying to remove the default